docs/book.toml

```
@@ -16,7 +16,7 @@ extra-watch-dirs = ["../crates/docs_preprocessor"]
 # The config is passed directly to the html renderer, so all config
 # options that apply to html apply to zed-html
 [output.zed-html]
-command = "cargo run -p docs_preprocessor -- postprocess"
+command = "cargo run -p docs_preprocessor -r -- postprocess"
```

Let's use the long version of the flag, as it is more self-explanatory:

```suggestion
command = "cargo run -p docs_preprocessor --release -- postprocess"
```